src/components/Codeblock.js

```diff
@@ -1,4 +1,5 @@
 import React, { useState, useContext } from "react"
+import PropTypes from "prop-types"
```

Comment:
Maybe I'm missing something, but I'm not sure this is required here. Testing locally without it seems to work just fine in the code blocks I checked.  Was there a reason for this?

Reply:
This is for developers, so as not to leave the Codeblock empty,  but i can remove it